src/shared/utils/formatPrice.test.ts

```diff
@@ -0,0 +1,275 @@
+import { describe, it, expect } from 'vitest';
+
+import { formatPrice } from './formatPrice';
+
+describe('formatPrice', () => {
+  const DEFAULT_THRESHOLD = 4;
```

Comment:
Can we add a quick blurb about the threshold.

// Number of zeros after the decimal that causes format to be condensed.
// i.e. condense if # zeros after decimal >= threshold.